osxiso

```diff
@@ -118,8 +119,8 @@ Select OS or Quit: "
 
     prep_build "$1"
 
-    # Create a blank ISO of 8121MB (7,93 GiB) with a Single Partition
-    hdiutil create -o "$temp/$1.cdr" -size 8121m -layout SPUD -fs HFS+J
+    # Create a blank ISO with a Single Partition
+    hdiutil create -o "$temp/$1.cdr" -size 13312m -layout SPUD -fs HFS+J
```

Comment:
I was able to use a size of `12700m` for a Big Sur 11.1 ISO (from app version 16.2.01)